internal/pkg/scaffold/helm/go_mod.go

```diff
@@ -149,15 +149,15 @@ require (
 	vbom.ml/util v0.0.0-20180919145318-efcd4e0f9787 // indirect
 )
 
-// Pinned to kubernetes-1.13.1
+// Pinned to kubernetes-1.14.1
 replace (
 	k8s.io/api => k8s.io/api v0.0.0-20181213150558-05914d821849
```

Comment:
All of these need updating. You can simply replace the versions with `kubernetes-1.14.1`.

Reply:
Done